.changeset/nervous-impalas-serve.md

```diff
@@ -0,0 +1,5 @@
+---
+"@fuel-ts/account": patch
```

Comment:
```
"@fuel-ts/account": minor
```

Isn't this a breaking change given we [changing](https://github.com/FuelLabs/fuels-ts/pull/3336/files#diff-df8d51395ea0431bdaeb7f953b65d98fc4fc0a4f2cce253f31a2b4c87c616c14R495) the response of `getTransactions` to return two less fields? i.e. the `id` and `transactionStatusFragment` ?

Reply:
But @maschad The [Provider.getTransactions](https://github.com/FuelLabs/fuels-ts/blob/9ebd6c224482a9b3c409409e9f0e14cb0de38224/packages/account/src/providers/provider.ts#L1685) has exactly the same response, not property was removed from it.

Reply:
I agree with you but if someone called `provider.operations.getTransactions()` the response would change and thus it's breaking.

Reply:
We made #3286 and #3296 breaking for the same reason.

Reply:
@maschad Oook I get you now.

I think this one went unnoticed https://github.com/FuelLabs/fuels-ts/pull/3309

Reply:
https://github.com/FuelLabs/fuels-ts/pull/3336/commits/fe146700202b4f338b73865c89b8abbe39552f75